backend/tests/integration/submissions/test_submissions_views.py

```diff
@@ -2277,3 +2278,49 @@ def test_new_institution_submission_with_empty_data(
 
     assert response.status_code == 400
     assert response.json["message"][0] == "Institution is missing a value or values."
+
+
+def test_new_journal_submission(
+    inspire_app,
+):
+    journal_data = {
+        "_collections": ["Journals"],
+        "$schema": "http://localhost:5000/schemas/records/journals.json",
+        "short_title": "RoMP",
+        "journal_title": {"title": "Reviews of Modern Physics"},
+        "control_number": 128,
+        "self": {'$ref': 'http://localhost:5000/api/journals/128'},
+    }
+    form_data = {
+        "short_title": "RoMP",
+        "journal_title": {"title": "Reviews of Modern Physics"},
+    }
+
+    user = create_user()
+    record = create_record("jou", data=journal_data)
+    
+    with inspire_app.test_client() as client:
+        login_user_via_session(client, email=user.email)
+        response = client.post(
+            "/submissions/journals",
+            content_type="application/json",
+            data=orjson.dumps({"data": form_data}),
+        )
+    assert response.status_code == 201
+    assert journal_data == record
```

Comment:
it doesn't make sense, you need to send the data in a submission and then test that the journal was created by the endpoint